pkg/proxy/iptables/proxier.go

```diff
@@ -157,6 +155,30 @@ type serviceInfo struct {
 	serviceLBChainName       utiliptables.Chain
 }
 
+// ClusterIP is part of interface.
+func (info *serviceInfo) ClusterIP() string {
+	return info.clusterIP.String()
+}
+
+// Port is part of interface.
+func (info *serviceInfo) Port() int {
+	return info.port
+}
+
+// Protocol is part of interface.
+func (info *serviceInfo) Protocol() api.Protocol {
+	return info.protocol
+}
+
+func (info *serviceInfo) String() string {
```

Comment:
Not sure if you wanted to add a comment here?

Reply:
Yes, we need it.

Reply:
Done